docs/guides/payments.rst

```diff
@@ -0,0 +1,226 @@
+.. _adding-payments:
+
+Payments
+========
+
+Integrating new Payment Gateway into Saleor
+-------------------------------------------
+
+We are using a universal flow, that each provider should fulfill, there are
+several methods that should be implemented.
+
+Your changes should live under the
```

Comment:
under the... namespace? Package? Not sure how it's called in Python?

Reply:
Module is good I think